semstr/convert.py

```diff
@@ -171,6 +171,34 @@ def to_sdp(passage, test=False, tree=False, mark_aux=False, **kwargs):
     return SdpConverter(mark_aux=mark_aux, tree=tree).to_format(passage, test=test, format=kwargs.get("format"))
 
 
+def from_ptb(lines, passage_id=None, return_original=False, **kwargs):
+#def from_ptb(passage, passage_id=None, return_original=False, **kwargs):
```

Comment:
Remove commented-out lines unless there's a reason to keep them.

Reply:
Removed.